debian/nfd.service

```diff
@@ -32,7 +32,10 @@ After=nfd-keygen.service
 [Service]
 EnvironmentFile=-/etc/default/nfd
 Environment=HOME=/var/lib/ndn
-ExecStart=/bin/sh -c 'LANG=C /usr/bin/nfd --config /etc/ndn/nfd.conf ${OPTIONS}'
+# FIXME: Needed to work around libboost bug
+# https://svn.boost.org/trac/boost/ticket/4688
+Environment=LANG=C
+ExecStart=/usr/bin/nfd --config /etc/ndn/nfd.conf ${OPTIONS}
```

Comment:
Accept the commit for after the demo and it should become pretty obvious if `nfd` starts crashing, I guess. I’m pretty sure this commit is correct though.